src/components/Buttons/Button.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+export default function Button() {}
```

Comment:
Is this going to be filled up in the next step?

Reply:
Yes, thanks. I forgot to delete ist.